runtime/dancebox/tests/common/mod.rs

```diff
@@ -150,7 +150,7 @@ pub fn set_parachain_inherent_data() {
     let relay_sproof = RelayStateSproofBuilder {
         para_id: 100u32.into(),
         included_para_head: Some(HeadData(vec![1, 2, 3])),
-        current_slot: (current_slot() * 2).into(),
+        current_slot: (current_slot()).into(),
```

Comment:
What is the reasoning behind this change?

Reply:
As we now have the same clocktime as the relay (6 seconds), we don't need to double the slot that we mock. Previously we were forced to do this because for each para-block (with 12 seconds blocktime), we were receiving a relay slot that was the double of the previous one.